pkg/proc/exec.go

```diff
@@ -0,0 +1,70 @@
+package proc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+
+	//nolint:depguard // TODO: Switch Run() over to dexec.
+	"os/exec"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/shellquote"
+)
+
+// Run will run the given executable with given args and env, wait for it to terminate, and return
+// the result. The run will dispatch signals as appropriate for the given platform (SIGTERM and SIGINT on Unix platforms
+// and os.Interrupt on Windows).
+func Run(ctx context.Context, exe string, args []string, env map[string]string) error {
+	cmd := exec.CommandContext(ctx, exe, args...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	cmd.Stdin = os.Stdin
```

Comment:
Just using `os` instead of taking a Cobra command as an argument.

https://github.com/telepresenceio/telepresence/pull/1864#discussion_r681514697